### ISSUE TITLE
The Volume OSD has blurry (clipped-off) fonts

### ISSUE THREAD

**Issue by codic12** [linuxmint/cinnamon#10060]
* Cinnamon 4.8.6
 * Distribution: Arch
 * Graphics hardware _Intel Iris Xe graphics, and Intel Intel Graphics 4000 with i915 and intel drivers respectively_
 * x86_64
 

**Issue**
The OSD slider for the volume often has a blurry font.

**Steps to reproduce**
Set the font to something with size 9 ( a simple Arch install with Cinnamon will get you this )
Set the OSD size to large under notifications (for the most noticeable result)
![image](https://user-images.githubusercontent.com/48339289/115974247-390ac900-a510-11eb-91d6-e988e398c9ed.png)
Notice the text.

This is reproducible under some other combos as well.

**Expected behaviour**
The text is crisp and sharp. (This is not related to the font and happens with several of them)

**Comment by brownsr** [comment]
Can you please confirm which theme you are using ?

**Comment by codic12** [comment]
Cinnamon's inbuilt shell theme. It seems to be present in other themes as well, but most noticable in that one.

**Comment by ItzSwirlz** [comment]
In the end of the day I think this depends on the theme

**Comment by codic12** [comment]
I don't believe so. To some extent it happens with every theme.

**Comment by Furvy** [comment]
Yeah... This is definitely an issue with Cinnamon on Arch. I opened a set of very detailed threads on this exact issue and begged for someone to take a look at it, but no results really came. Not sure why nobody will address this?

https://github.com/linuxmint/cinnamon/issues/9758

https://github.com/linuxmint/cinnamon/pull/9811

It is directly an issue with how osdWindow.js is programmed. It does not work perfectly with resolutions other than 1920x1080 and I would absolutely love to just program the issue myself, but I am not going to learn JavaScript just for this...

**Comment by codic12** [comment]
Hmm, what ended up happening to that PR you made? Might give it a go locally, maybe have a pacman hook to overwrite that file

**Comment by Furvy** [comment]
codic12, thank you very much for your consideration, but if I am being brutally honest, I have no idea what I am doing and I simply just changed a few dimensions in that file out of pure desperation. The core issue is that this guy added the percentage label code with no regards to how it worked with the rest of the framework of how osdWindow.js is visually structured:

https://github.com/linuxmint/cinnamon/commit/ce4c8d266b09df0d752cd1276c7cbd835990e86e

The pull request was never approved or fixed by clefebvre despite him approving the above botched code, so I closed it a month later because it was ignored. I am so frustrated by this that I dumped Cinnamon and switched to Xfce.

**Comment by codic12** [comment]
Little things like this bother me too, I'm on Budgie for the time being.Very curious. I think the font-size: 1.2em may be the problem; I might just disable Cinnamon's volume OSD and media keys entirely and write my own script to do it with notify-send. but it doesn't bother me too much, dunno if it's worth the effort.
